### DISCUSSION TITLE
defaultViewport or window-size in browserless v2

### DISCUSSION THREAD

**Discussion by masterwishx** [browserless/chrome#3910] [Q&A] [answered]
should we use defaultViewport or window-size in browserless v2 ?

`ws://192.168.0.199:3012/chrome?token=mytoken&launch={"defaultViewport":{"height":1080,"width":1920},"headless":false,"stealth":true}&blockAds=true`

or `ws://localhost:3000?token=ABCD&launch={"args":["--window-size=1920,108"]}`

**Comment by joelgriffith** [comment, marked as answer]
I think _both_ would work. The first one you mentioned is going to apply it to all pages whereas the latter applies it to chromes window (yes they both have their own setting). I'd recommend combining them both into one URL

**Reply by brendon** [reply]
By "all pages" and "chromes window" do you mean that the first option is global to all browsers in Browserless and not just Chrome, and that the second configuration only applies to the Chrome browser?
